presentation/tutorial.jl

```diff
@@ -0,0 +1,815 @@
+### A Pluto.jl notebook ###
+# v0.17.7
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 19b71044-a0c8-468a-8e13-91b17d336497
+import Pkg; Pkg.activate
+
+# ╔═╡ 599711e7-ee51-4ca0-94ac-8c1a67f6ac8d
+using CSV, Microbiome, Microbiome.SparseArrays, BiobakeryUtils, BiobakeryUtils.Conda, Random;
+
+# ╔═╡ 84470294-5a17-469e-890a-5e2b9f777dbb
+using Downloads: download
```

Comment:
i don't think you need to show these as part of the tutorial? maybe you can just hide them?

Reply:
It might be important to have them since they are necessary to run the packages... @kescobo what do you think?